native/src/main/java/io/ballerina/stdlib/http/api/logging/util/LogLevelMapper.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Better to update the copyright statement consistently.

Reply:
Theses files are just moved from lang repo. That is why I did not change anything. I can change them.

Reply:
Yeah, Since those are not newly wrote files, let's keep it as it is.